server/src/main/java/io/crate/metadata/cluster/AlterTableClusterStateExecutor.java

```diff
@@ -257,47 +245,24 @@ public static Map<String, Object> settingsToMapping(Settings settings) {
 
     static ClusterState updateTemplate(ClusterState currentState,
                                        RelationName relationName,
-                                       Settings newSetting,
+                                       Settings newSettings,
                                        Map<String, Object> newMapping,
-                                       BiConsumer<String, Settings> settingsValidator,
-                                       IndexScopedSettings indexScopedSettings) throws IOException {
+                                       IndexScopedSettings indexScopedSettings) {
 
         String templateName = PartitionName.templateName(relationName.schema(), relationName.name());
         IndexTemplateMetadata indexTemplateMetadata = currentState.metadata().templates().get(templateName);
         IndexTemplateMetadata newIndexTemplateMetadata = DDLClusterStateHelpers.updateTemplate(
             indexTemplateMetadata,
             newMapping,
             Collections.emptyMap(),
-            newSetting,
+            newSettings,
             indexScopedSettings
         );
 
         final Metadata.Builder metadata = Metadata.builder(currentState.metadata()).put(newIndexTemplateMetadata);
         return ClusterState.builder(currentState).metadata(metadata).build();
     }
 
-    private static void validateSettings(String name,
```

Comment:
was not used, I tried to use it and we have test failures, since the validation is now taking place elsewhere.